embedly/embedly.py

```diff
@@ -27,27 +29,29 @@ def get_cached_url(url):
 
 def set_cached_url(url, data):
     redis_client.set(url, json.dumps(data))
-    redis_client.expire(url, REDIS_TIMEOUT)
+    redis_client.expire(url, app.config['REDIS_TIMEOUT'])
 
 
 def get_urls_from_embedly(urls):
     params = '&'.join([
-        'key={}'.format(EMBEDLY_KEY),
+        'key={}'.format(app.config['EMBEDLY_KEY']),
         'urls={}'.format(','.join([urllib.quote_plus(url) for url in urls])),
     ])
 
-    request_url = '{base}?{params}'.format(base=EMBEDLY_URL, params=params)
+    request_url = '{base}?{params}'.format(
+        base=app.config['EMBEDLY_URL'],
+        params=params,
+    )
+
     response = requests.get(request_url)
```

Comment:
Better capture possible exceptions raised by requests.get, for example, `Timeout`, `ConnectionError`, and `TooManyRedirects`. Or capture `equests.exceptions.RequestException` to bail.